README.md

```diff
@@ -1,13 +1,140 @@
 `next-crud` is a helper library that creates CRUD API routes with one simple function based on a Prisma model for Next.js.
 
-### Installation
+# Installation
 
 `yarn add next-crud`
 
-### Usage
+or
 
-TODO
+`npm i -S next-crud`
 
-### Example
+# Usage
+
+- Create a file under `pages/api/<modelName>` called `[[...someName]].ts`. You can also have 2 files called `index.ts` and `[id].ts` under that folder, both files will have to contain the same content.
```

Comment:
>  You can also have 2 files called `index.ts` and `[id].ts` under that folder, both files will have to contain the same content.

Why ?

Reply:
I think we need to guide more on the "someName" part, what does it mean ?

Reply:
Because we can have both files notation, but if we use 2 files we still need to have the next crud handler in each one